run_pylint.sh

```diff
@@ -34,6 +34,7 @@ INIT_HOOK="
 AP = os.environ.get('AUTOTEST_PATH', '/usr/local/autotest')
 sys.path.append(os.path.abspath(AP + '/..'))
 sys.path.append(os.path.abspath('.'))
+sys.path.append(os.path.abspath('dockertest'))
```

Comment:
It will be _much_ easier to do w/in a minor release where upsetting the API (namespace) is more kosher.  Less duct-tape will be used (and we can kill the damn ``xceptions`` module - it's nothing but trouble).